templates/main.html

```diff
@@ -57,6 +57,15 @@ <h3>
                 />
               </a>
             {{/github}}
+            {{#gitlab}}
+              <a href="https://gitlab.com/{{gitlab}}">
+                <img
+                  src="images/logos/gitlab.png"
```

Comment:
If you are storing a copy in the repository, please ensure that the licensing of the artwork is complied with.

Reply:
It's under [CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) so yea we have to provide attribution. Should I put this in the footer?

Reply:
Storing it in the repo means it needs to be attributed in the relevant directory of the repo, as it isnt covered by the repo license.

Reply:
a README in that directory would be useful.  Also for CC licenses, the full text of the license isnt necessary.  A link is acceptable, and a link to the license file in their repo is really good, as it allows the reader to find the original, which is the intent of CC.

Reply:
So instead of having the full text in the LICENSE file, just have a link in the LICENSE file? And show which images are under which licenses in a README file?